includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -101,10 +121,34 @@ public function get_settings(): WP_REST_Response {
 	public function update_settings( WP_REST_Request $request ) {
 		$this->update_is_wcpay_enabled( $request );
 		$this->update_enabled_payment_methods( $request );
+		$this->update_is_digital_wallets_enabled( $request );
```

Comment:
Should these also be wrapped by the feature flag check, in case `$this->digital_wallets_gateway` is `null`?
Not sure if we care too much, since this is going to be refactored soon to _not_ use the `Digital_Wallets_Payment_Gateway` gateway anyways 😅